### ISSUE TITLE
On logout Loot Tracker deletes all loot tracker data tied to accountId tied to deleted session

### ISSUE THREAD

**Issue by deathbeam** [runelite/runelite#7784]
This seems very wrong.

**Comment by deathbeam** [comment]
Looks like this is the issue:

```sql
FOREIGN KEY (accountId) REFERENCES sessions(user) ON DELETE CASCADE
```

**Comment by deathbeam** [comment]
Fixed in 55395b67084747457cd3b7057da9bc937afa2802
